src/components/Recipes/RecipeButton.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { Button } from "react-bootstrap";
+import { Link } from "react-router-dom";
+
+export const RecipeButton = ({ btnText, btnLink }) => {
+  return (
+    <>
+      <Button className="button-main">
+        <Link to={btnLink}>{btnText}</Link>
```

Comment:
if you move the link outside of the button element it won't apply the default styling

Reply:
I have changed this button in the next branch so I won't change here.